src/signals/heredoc_signal.c

```diff
@@ -0,0 +1,34 @@
+#include <signal.h>
+#include <stdio.h>
+#include <readline/readline.h>
+#include <readline/history.h>
+#include <libft.h>
+#include <defines.h>
+#include <output/prompt.h>
+
+int	*heredoc_sigint(void)
+{
+	static int	sig_interupt = 0;
+
+	return (&sig_interupt);
+}
+
+static void	heredoc_signals(int code)
+{
+	int	*signal;
+
+	(void)code;
+	signal = heredoc_sigint();
+	*signal = 667;
```

Comment:
what does 667 mean?
I think the number 1 on  rl_done can become TRUE and 667 can be a const with a nice name hehe